lib/route.js

```diff
@@ -344,6 +345,13 @@ function buildRouting (options) {
       }
     }
 
+    if (req.headers['keep-alive']) {
```

Comment:
Should we monitor the connection header too?

Reply:
No. That would extra processing up front and extra timers per request.

Reply:
Oh, I misread that. You're asking if we should also check that `req.headers.connection === 'keep-alive'`. I think you're right, it should be checked. Re-reading the spec, `keep-alive: <settings>` is optional.